scripts/exrrfs_nonvar_reflobs.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091,SC2154,SC2086,SC2068
+declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LINENO}]: '
+set -x
+cpreq=${cpreq:-cpreq}
+
+cd "${DATA}" || exit 1
+#
+#-----------------------------------------------------------------------
+#
+# Extract from CDATE the starting year, month, day, and hour of the
+# forecast.  These are needed below for various operations.
+#
+#-----------------------------------------------------------------------
+#
+#
+YYYY=${CDATE:0:4}
+MM=${CDATE:4:2}
+DD=${CDATE:6:2}
+HH=${CDATE:8:2}
+
+#
+#-----------------------------------------------------------------------
+#
+# link or copy background files
+#
+#-----------------------------------------------------------------------
+#
+meshgriddir="${FIXrrfs}"/meshes
+echo "INFO: meshgriddir is $meshgriddir"
+cp "${meshgriddir}"/"${MESH_NAME}".grid.nc mesh.nc
+
+#
+#-----------------------------------------------------------------------
+#
+# link/copy observation files to working directory
+#
+#-----------------------------------------------------------------------
+#
+
+obs_appendix=grib2
+if [[ ${MACHINE} == "WCOSS2" ]]; then
+  obs_appendix=grib2.gz
+fi
+NSSL=${OBSPATH_NSSLMOSIAC}
+
+mrms="MergedReflectivityQC"
+
+#
+#-----------------------------------------------------------------------
+#
+# Link to the MRMS operational data
+#
+#-----------------------------------------------------------------------
+#
+
+if [ -s filelist_mrms ]; then
+  rm -f filelist_mrms
+fi
+for (( j=0; j < 4; $((j=j+1)) )); do
+  min=$( printf %2.2i ${j} )
+  echo "Looking for data valid:${YYYY}-${MM}-${DD} ${HH}:${min}"
+  if [[ -e filelist_mrms ]]; then
+    break
+  fi
+  s=0
+  while [[ $s -le 59 ]]; do
+    ss=$(printf %2.2i ${s})
+    nsslfile="${NSSL}/*${mrms}_00.50_${YYYY}${MM}${DD}-${HH}${min}${ss}.${obs_appendix}"
+    if [ -s ${nsslfile} ]; then
+      echo "Found ${nsslfile}"
+      nsslfile1="*${mrms}_*_${YYYY}${MM}${DD}-${HH}${min}*.${obs_appendix}"
+      numgrib2=$(find ${NSSL}/${nsslfile1} -maxdepth 1 -type f | wc -l)
+      echo "Number of GRIB-2 files: ${numgrib2}"
+      if [ "${numgrib2}" -ge 10 ] && [ ! -e filelist_mrms ]; then
```

Comment:
suggest changing to:
`if (( numgrib2 >= 10)) && [[ ! -s filelist_mrms ]]; then`